src/sitemap.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>https://queridodiario.ok.org.br/</loc>
+        <changefreq>monthly</changefreq>
+        <priority>1.0</priority>
+    </url>
+    <url>
+        <loc>https://queridodiario.ok.org.br/tecnologia</loc>
+        <changefreq>monthly</changefreq>
+        <priority>0.7</priority>
+    </url>
+    <url>
+        <loc>https://queridodiario.ok.org.br/pesquisa?term=term</loc>
+        <changefreq>daily</changefreq>
+        <priority>0.9</priority>
+    </url>
+    <url>
+        <loc>https://queridodiario.ok.org.br/sobre</loc>
+        <changefreq>monthly</changefreq>
+        <priority>0.5</priority>
+    </url>
+    <url>
+        <loc>https://queridodiario.ok.org.br/educacao</loc>
+        <changefreq>monthly</changefreq>
+        <priority>0.8</priority>
+    </url>
+    <url>
+        <loc>https://queridodiario.ok.org.br/educacao/busca</loc>
+        <changefreq>monthly</changefreq>
+        <priority>0.9</priority>
+    </url>
+    <url>
+        <loc>https://queridodiario.ok.org.br/educacao/sobre</loc>
+        <changefreq>monthly</changefreq>
+        <priority>0.5</priority>
+    </url>
+    <url>
+        <loc>https://queridodiario.ok.org.br/blog</loc>
+        <changefreq>weekly</changefreq>
+        <priority>0.8</priority>
+    </url>
+    <url>
+        <loc>https://queridodiario.ok.org.br/apoie</loc>
+        <changefreq>monthly</changefreq>
+        <priority>0.7</priority>
+    </url>
+    <!-- Adicionando pdfs para teste de indexação e comportamento na lista de resultados dos buscadores. -->
+    <url>
+        <loc>https://querido-diario.nyc3.cdn.digitaloceanspaces.com/2611606/2019-03-16/d923d35d533f0bb7c80f7ee7c887e10e0136d295.pdf</loc>
+        <lastmod>2024-03-07</lastmod>
+        <changefreq>yearly</changefreq>
+    </url>
+    <url>
+        <loc>https://querido-diario.nyc3.cdn.digitaloceanspaces.com/3106200/2007-05-03/9fefed7185f514e1a94645d38b9bd1432a7d5eb6</loc>
+        <lastmod>2024-03-07</lastmod>
+        <changefreq>yearly</changefreq>
+        <priority>0.5</priority>
+    </url>
+    <url>
+        <loc>https://querido-diario.nyc3.cdn.digitaloceanspaces.com/2103901/2018-10-11/a39217e5a83507f75e2750d9e66d0035ec755e2b.pdf</loc>
+        <lastmod>2024-03-07</lastmod>
+        <changefreq>yearly</changefreq>
+        <priority>0.5</priority>
+    </url>
+    <url>
+        <loc>https://querido-diario.nyc3.cdn.digitaloceanspaces.com/4205407/2019-10-18/118aecb85e355ac01660d29916bfd172971b22df.pdf</loc>
+        <lastmod>2024-03-07</lastmod>
+        <changefreq>yearly</changefreq>
+        <priority>0.5</priority>
+    </url>
```

Comment:
Pode retirar esses documentos do sitemap?

Reply:
Esses eram arquivos de teste que ia aproveitar para subir, apenas para observarmos como são indexados e como aparecem nos resultados do Google, como referência. Não são essenciais e podem ser removidos, se for a recomendação, testados em outra oportunidade.